src/model/reddit.ts

```diff
@@ -651,7 +651,6 @@ export async function getMine(data: IAPIData): Promise<IGenericResult<IListing<I
                 url += "limit=" + data.limit + "&";
             }
             url += "raw_json=1";
-            if(!data.cookie){ data.cookie = ""; }
```

Comment:
Why did you remove this?
As far as I remember this fixed a bug where the old session was still used.

Reply:
I'm pretty sure I didn't remove that, and I don't think vscode would delete it for no reason so I might have made a bad manipulation.